### PR TITLE
Angular Accordion - add `aria-expanded` and change element

### PR DESCRIPTION
**Pull request by bobwise**
## What does this PR do?
- Adds the `aria-expanded` attribute to the Angular Accordion Item component.
- Changes the root element from `<a>` to `<button>`.
- Adds some unit tests

### Associated Issue
Fixes https://github.com/sparkdesignsystem/spark-design-system/issues/2939

### Code
 - [X] Build Component in Angular
 - [X] Unit Testing in Angular with `npm run test` in `angular/` (100% coverage, 100% passing)

### Accessibility
- [X] New changes abide by [accessibility requirements](https://sparkdesignsystem.com/docs/accessibility)

### Browser Testing (current version and 1 prior)
  - [X] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)

### CODE REVIEW

**Comment by bobwise** [comment]
Do not merged until the button CSS is available.

**Comment by afebbraro** [comment]
Awesome! Will hold off on approval until CSS is ready in the dependent PR.

**Comment by bobwise** [comment]
This needs a little more CSS work on top of RV's change. I'll update when it's ready for review.

File: html/components/_accordions.scss [hunk 1]
```diff
@@ -39,6 +39,9 @@
   display: flex;
   padding: $sprk-accordion-summary-padding;
   text-decoration: none;
+  text-align: left;
+  border: none;
```
Comment (afebbraro) [inline html/components/_accordions.scss]: lets make border and background color both variables

File: html/settings/_settings.scss [hunk 2]
```diff
@@ -1367,6 +1367,12 @@ $sprk-accordion-max-width: 53.125rem !default;
 $sprk-accordion-border: $sprk-divider-border !default;
 /// The font color of the Accordion item summary.
 $sprk-accordion-summary-color: $sprk-black !default;
+/// The background color of the Accordion item summary
+$sprk-accordion-summary-background-color: transparent !default;
+/// The text-align value of the Accordion item summary
+$sprk-accordion-summary-text-align: left !default;
+/// The border of the Accordion item summary
```
Comment (afebbraro) [inline html/settings/_settings.scss]: ``
```suggestion
/// The border of the Accordion item summary.
```

File: html/settings/_settings.scss [hunk 3]
```diff
@@ -1367,6 +1367,12 @@ $sprk-accordion-max-width: 53.125rem !default;
 $sprk-accordion-border: $sprk-divider-border !default;
 /// The font color of the Accordion item summary.
 $sprk-accordion-summary-color: $sprk-black !default;
+/// The background color of the Accordion item summary
+$sprk-accordion-summary-background-color: transparent !default;
+/// The text-align value of the Accordion item summary
```
Comment (afebbraro) [inline html/settings/_settings.scss]:
```suggestion
/// The text-align value of the Accordion item summary.
```

File: html/settings/_settings.scss [hunk 4]
```diff
@@ -1367,6 +1367,12 @@ $sprk-accordion-max-width: 53.125rem !default;
 $sprk-accordion-border: $sprk-divider-border !default;
 /// The font color of the Accordion item summary.
 $sprk-accordion-summary-color: $sprk-black !default;
+/// The background color of the Accordion item summary
```
Comment (afebbraro) [inline html/settings/_settings.scss]:
```suggestion
/// The background color of the Accordion item summary.
```

**Pull request merged**